app/views/activities/_activity.html.erb

```diff
@@ -9,16 +9,27 @@
 
   <td class="activity-event-and-subject">
     <span class="activity-event">
-      <%= t(
-        ".event.#{activity.event}_html",
-        assignment: link_to(
-          activity.result.name,
-          manage_results_assignment_path(activity.result.assignment_id)
-        ),
-      ) %>
+      <% if activity.item_type == "Result" %>
```

Comment:
What do you think of breaking these conditionals out into their own partials? So we could do something like:

```
render "#{activity.item_type.downcase}_activity", item: item
```